.travis.yml

```diff
@@ -13,6 +13,19 @@ node_js:
 - '6'
 - '7'
 
+cache:
+  directories:
+    - node_modules
+
+install:
+  - npm install
+
+script:
+  - npm run-script coverage
+
+notifications:
+  email: false
+
 matrix:
   include:
   - node_js: '0.10'
```

Comment:
I agree. I believe @mreinstein added this originally from [this commit](https://github.com/alexa-js/alexa-verifier-middleware/commit/508821970ee7720a0fc3e97bdb11c78ba9cc4fd6). He might be able to explain why.